Frontend/src/pages/ExploreSystems.js

```diff
@@ -486,10 +486,38 @@ function ExploreSystems() {
                                     >
                                         Rational Preperiodic Points
                                     </span>
-                                    <ul className="nested"></ul>
+                                    <ul className="nested">
+                                        <li>
+                                            <input
+                                                type="number"
+                                                style={textBoxStyle}
+                                                value={filters.rationalPreperiodicCardinality || ''}
+                                                onChange={(e) => handleTextChange("rationalPreperiodicCardinality", e.target.value)}
+                                            />
+                                            <label>Cardinality</label>
+                                        </li>
+                                        <li>
+                                            <input
+                                                type="number"
+                                                style={textBoxStyle}
+                                                value={filters.rationalPreperiodicComponents || ''}
+                                                onChange={(e) => handleTextChange("rationalPreperiodicComponents", e.target.value)}
```

Comment:
I think this is a solid fix for the alignment for the sub groups of Rational Preperiodic Components.